### ISSUE TITLE
[Project] Ambermoon Advanced

### ISSUE THREAD

**Issue by Pyrdacor** [Pyrdacor/Ambermoon.net#179]
Ambermoon Advanced is my next project.
There is already a GitHub page but nothing more than e readme yet: https://GitHub.com/Pyrdacor/Ambermoon-Advanced

I chose it before the third part of the trilogy as I can test some ideas, build some tools and finish something "new" without having to pay someone to create completely new graphics and music.

So the third part of the trilogy is not dead, it is only delayed a bit. But I hope to be able to use parts of this project (tools, ideas, findings) for the sequel as well. So the delay is not wasted time. It is kind of a research and preparation part for the sequel. I would have to create tools and editors for it anyway.

I also chose to not create a Ambermoon.net-only Mod. Sure I could add fancy stuff like completely new game features but I want to start with a core which is compatible to the Amiga version.

But of course I can add more features later to Ambernoon.net which then are not available for Amiga unfortunately.

So this is the agenda:

- Ambermoon Advanced (Amiga and PC)
- Additional features only for Ambermoon.net (contains Ambermoon Advanced stuff as well)
- Third part of the trilogy

**Comment by Pa9an** [comment]
Great news :)
But I presume (hope) that "PC" will mean Windows + Linux + Mac ? No just Windows only?

**Comment by a1exh** [comment]
> I presume (hope) that "PC" will mean Windows + Linux + Mac ? No just Windows only?

I am sure it is, @Pyrdacor has gone to so much trouble to support Linux and MacOS I cannot imagine he will stop now. Although I am sure he will need help testing on each platform.

**Comment by Pyrdacor** [comment]
Of course. Basically it is a mod to the original Amiga game data and as Ambermoon.net can directly use it, you can play it with Ambermoon.net, which is available for Linux and Mac as well. ;)

**Comment by Pa9an** [comment]
I was 99,9% sure to get this kind of answer, but that missing 0,1% was nagging me ;-) Thanks and I am looking forward to that.
I'm available for Linux testing in case you will need it at some point :-)

**Comment by Pyrdacor** [comment]
A small project update.

I adjusted all the characters and added my first completely new NPC with some interactions. It works fine on the Amiga as well.

Now I will try to add my first new item. To make this easier I started to write an item editor. As items are stored inside the executables, it will take care of offset adjustments inside the code, hunk size adjustments and writing to both executables as well (maybe I will add a transfer function to ease transfer to other languages as well).

**Comment by Pyrdacor** [comment]
I added 2 new places to Spannenberg. Both will have new 2D maps. The first is already finished. There is also a new NPC on that new map who can do some cool stuff dependent on your game progress and gold.

The effects are not implemented yet but nearly all of the conversation etc.

My first new item was also added a while ago. I wrote a small CLI item editor for that.

Adding NPCs, maps and quests is quiet hard and a lot of work. :/

**Comment by Pyrdacor** [comment]
The first episode "Mysteries of the sea" was released today. Have a look at https://GitHub.com/Pyrdacor/Ambermoon-Advanced.
